e2e/src/completion/completion_macros.spec.ts

```diff
@@ -20,7 +20,6 @@ suite('Should suggest macros completions', () => {
 
     // act
     const actualCompletionList = await triggerCompletion(docUri, new Position(0, 17));
-    console.log(actualCompletionList);
```

Comment:
Why didn't you add json logs here as we discussed?

Reply:
Json logs could take more space in output.